api/src/main/java/org/pmiops/workbench/db/dao/UserCodeOfConductAgreementDao.java

```diff
@@ -1,11 +1,7 @@
 package org.pmiops.workbench.db.dao;
 
-import java.util.List;
-import org.pmiops.workbench.db.model.DbUser;
 import org.pmiops.workbench.db.model.DbUserCodeOfConductAgreement;
 import org.springframework.data.repository.CrudRepository;
 
 public interface UserCodeOfConductAgreementDao
-    extends CrudRepository<DbUserCodeOfConductAgreement, Long> {
-  List<DbUserCodeOfConductAgreement> findByUserOrderByCompletionTimeDesc(DbUser user);
-}
+    extends CrudRepository<DbUserCodeOfConductAgreement, Long> {}
```

Comment:
I'm assuming this dao is still used in the codebase for crud operations such save, delete, etc...?

Reply:
yes ... although I'm not sure it's still needed.  Let me test this idea a bit.

Reply:
We can indeed remove it.  Also found and fixed a bug in the process: #6262